RTN-002.tex

```diff
@@ -57,66 +59,71 @@
 different experience and expertise backgrounds, scientific motivations, and accessibility needs.
-Four user profiles for each of the Notebook, Portal, and API aspects of the RSP have been drafted along
-the axes of user experience and science motivation.
-Two additional user profiles have been drafted: one to represent integrated use of the Notebook and Portal aspects,
-and one to represent accessibility needs.
 
-These fourteen user profiles are subject to change.
+\textbf{Analysis use cases} are designed to represent the variety of types of analysis that
+users will do with the RSP, such as subsetting, visualization, model fitting, and reprocessing.
+These use cases help to define the limitations of the RSP as well, and identify analysis
+that requires external resources (independent data access centers; IDACs).
 
-\textbf{Use cases} for the support of scientists using the Rubin software and data products have been drafted
-in a format defined by model-based systems engineering (MBSE) practices.
-The seven use cases represent issues and support needs that involve, e.g., trouble with scientific analysis,
+\textbf{Support use cases} are designed to provide workflow examples of Rubin's issue resolution
+process, to illuminate how users report problems and get support, and how staff from across
+Rubin participate in user support.
+These use cases include, e.g., trouble with scientific analysis,
 instrumentation faults, software bugs.
 
-These seven use cases are subject to change.
-
 \clearpage
-\section{User Profiles for Science Validation of the Rubin Science Platform}
+\section{User Profiles}
 
-\clearpage
-\subsection{Portal Aspect}
-\input{up_portal.tex}
+\input{user_profiles.tex}
 
-\clearpage
-\subsection{Notebook Aspect}
-\input{up_notebook.tex}
 
 \clearpage
-\subsection{Web Application Programming Interface (API) Aspect}
-\input{up_web_api.tex}
+\section{Analysis Use Cases}
 
-\clearpage
-\subsection{Integrated Aspects}
-\input{up_integrated_aspects.tex}
+These analysis use cases represent general things that users want to do in the Rubin Science Platform,
+using specific scientific examples.
+They are not 1:1 related to the user profiles above, although some of the more advanced analysis
+use cases, for example, would be much more likely to be done by a heavy user.
 
-\clearpage
-\subsection{Accessibility}
-\input{up_accesibility.tex}
+The analysis use cases progress from ``simple" to ``more complicated".
+These cases, along with those in \citeds{dmtn-086}, will serve as the basis for usability tests and the CST's process for science validation of
+the RSP.
+
+\subsection{Rubin Science Platform}
+
+\input{analysis_rsp.tex}
+
+\subsection{Independent Data Access Centers (IDACs) and other computational facilities}
+
+\input{analysis_idacs.tex}
+
+\subsection{Alert Brokers}
+
+\input{analysis_brokers.tex}
 
 \clearpage
-\section{Use Cases}
+\section{Support Use Cases}
```

Comment:
Section 4.4 and later have not been updated to the new template.

Reply:
Updated now, sorry for missing this the first time!

Reply:
No worries! Confirmed new template applied to support use cases.